.travis.yml

```diff
@@ -34,6 +36,14 @@ matrix:
       env: PUPPET_GEM_VERSION="~> 3.3.0"
     - rvm: 2.1.0
       env: PUPPET_GEM_VERSION="~> 3.4.0"
+    - rvm: 1.8.7
+      env: PUPPET_GEM_VERSION="~> 4.2.0"
+    - rvm: 1.8.7
+      env: PUPPET_GEM_VERSION="~> 4.3.0"
```

Comment:
please add explicit support for Puppet v4.4 like this:
<pre>
    - rvm: 1.8.7
      env: PUPPET_GEM_VERSION="~> 4.4.0"
</pre>